db/schema.rb

```
@@ -12,783 +10,803 @@
+    t.string "self_help_group_discharge"
+    t.string "arrests_past_30days_admission"
+    t.string "arrests_past_30days_discharge"
+    t.bigint "client_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.string "provider_gateway_id"
+    t.index ["client_id"], name: "index_episodes_on_client_id"
+    t.index ["episode_key"], name: "index_episodes_on_episode_key", unique: true
+  end
+
+  create_table "ite_client_records", force: :cascade do |t|
```

rubocop (1.69.1)

[convention] Prefer single-quoted strings when you don't need string interpolation or special symbols.

(Style/StringLiterals)